src/main/java/Task.java

```diff
@@ -0,0 +1,34 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
+    public Task(String description) {
+        this.description = description;
+        this.isDone = false;
+    }
+
+    public String getName() {
+        return description;
+    }
+
+    public boolean isDone() {
+        return isDone;
+    }
+
+    public void setName(String desc) {
+        this.description = desc;
```

Comment:
Perhaps standardising whether to use the `this` keyword will help to improve readability.